README.md

```diff
@@ -1,2 +1,31 @@
 # Capstone-1
 My first capstone project at Microverse
+
```

Comment:
- [ ] I think, we should give some proper name and description about the project.

Reply:
Such as?